torch/_functorch/_aot_autograd/schemas.py

```diff
@@ -181,8 +181,8 @@ class SubclassCreationMeta:
     # We need to keep them around along with outer_size / outer_stride to plumb them
     # into __tensor_unflatten__
     attrs: Dict[str, Union["SubclassCreationMeta", None]]
-    outer_size: List[int]
-    outer_stride: List[int]
+    outer_size: List[Union[int, torch.SymInt]]
```

Comment:
I think I was expecting the `make_runtime_safe()` function below to be updated to also "scrub" SymInts from `outer_size/stride` here? Since we no longer need them at runtime

(cc / fyi  @jamesjwu )